javascript/javascript3/week3/canvas_main.js

```diff
@@ -0,0 +1,42 @@
+//Lets make some art using classes
+//To follow the mouse - optional
+const body = document.querySelector("body");
+const myCanvasToDrawCircle = document.getElementById("canvasId");
+const ctx = myCanvasToDrawCircle.getContext("2d");
+class Circle {
+  constructor(x, y, r, startAngle, endAngle, fillColor) {
+    ctx.beginPath();
+    ctx.arc(x, y, r, startAngle, endAngle);
+    ctx.fillStyle = fillColor;
+  }
+  draw() {
+    ctx.stroke();
+    ctx.fill();
+  }
+}
+
+document
+  .getElementById("canvasId")
```

Comment:
you already have the const storing he value myCanvasToDrawCircle, to save some repetition you could use it here, or simply listen for an event on the document/window itself as your canvas is covering the whole screen.